tycho-plugins/repository-utils/pom.xml

```diff
@@ -71,6 +71,11 @@
 			<artifactId>git-commit-id-plugin</artifactId>
 			<version>2.1.13</version>
 		</dependency>
+		<dependency>
+		    <groupId>org.tukaani</groupId>
```

Comment:
this seem to not have any OSI compatible license - just public domain, which is not open source by default.
This needs approval by legal before we can use it.